app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue

```diff
@@ -15,7 +15,7 @@
           :key="notificationItem.id"
           @click="() => onClickNotification(notificationItem)"
         >
-          <td>
+          <td class="text-truncate">
```

Comment:
@iamsivin Can you please post a screenshot after this change?

Reply:
@pranavrajs Same as above. This issue only occurs in links or a sentence with no space. Not showing 

<img width="1023" alt="image" src="https://user-images.githubusercontent.com/64252451/178902862-f55cee60-40c0-4ef2-b6c7-e3497e0f7fcb.png">